samples/basic/threads/README.rst

```diff
@@ -0,0 +1,28 @@
+.. _96b_carbon_multi_thread_blinky:
+
+Basic Thread Example
+####################
+
+Overview
+********
+
+This example demonstrates spawning of multiple threads using K_THREAD_DEFINE.
+
+The example works by spawning  three threads. The first two threads control a
+separate LED. Both of these LEDs (USR1 and USR2) have their individual loop
+control and timing logic defined by separate functions.
+
+The third thread, uart_out(), sends out messages on the uart using printk().
+
+- blink1() controls the USR1 LED that has a 100ms sleep cycle
+- blink2() controls the USR2 LED that has a 1000ms sleep cycle
+
+Each thread is then defined at compile time using K_THREAD_DEFINE.
+
+Building
+********
+
+.. code-block:: console
+
+    $ cd samples/basic/threads
+    $ make
```

Comment:
Nice start... how do you run the sample?  Will it only run on a 96b_carbon (worth mentioning if so).

Reply:
Like other samples, you would need to change the pin numbering for other boards, I'll mention it.

Reply:
@ric96, Thanks for coming up with the updated version. And nope, not really, user doesn't need to change anything, the sample just needs to use <board.h> and LED0_GPIO_PORT, LED0_GPIO_PIN, etc. it defines. See basic/blinky for an example of writing portable hw-related sample.

Reply:
@ric96 : Ping. Is the above gives enough info or more details would be helpful?

Reply:
@pfalcon seems enough to me, I've covered the "what it does" part under Overview. And i do not want to include the "how to flash" part as it varies from board to board.

Reply:
Oops, I missed that you've already rewritten it to use board.h, that's what I meant. Will try it tomorrow.